reqrespupdate/README.md

```diff
@@ -0,0 +1,49 @@
+# Request/Response Sample with Update-Based Responses
+
+This sample demonstrates how to send a request and get a response from a Temporal workflow via an update.
+
+[Update](https://docs.temporal.io/workflows#update) is a new feature available for preview on [Temporal Server v1.21](https://github.com/temporalio/temporal/releases/tag/v1.21.0).
```

Comment:
May need some internal reminder to come update this once it's out of preview (including some of the verbiage below like "once released")